src/main/scala/io/sdkman/changelogs/GraceMigrations.scala

```diff
@@ -0,0 +1,59 @@
+package io.sdkman.changelogs
+
+import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
+import com.mongodb.client.MongoDatabase
+import org.bson.Document
+
+@ChangeLog(order = "084")
+class GraceMigrations {
+
+  val CandidateName = "grace"
+
+  @ChangeSet(
+    order = "001",
+    id = "001_add_grace_candidate",
+    author = "rainboyan"
+  )
+  def migration001(implicit db: MongoDatabase) : Candidate = {
+    Candidate(
+      candidate = CandidateName,
+      name = "Grace Framework",
+      description =
+        "An open-source and powerful One-Person web application Framework to help developers quickly build Spring Boot applications written in the Groovy programing language. Grace is a forked of Grails 5 that started development in early 2022.",
+      websiteUrl = "https://graceframework.org"
+    ).insert()
+  }
+
+  @ChangeSet(
+    order = "002",
+    id = "002_add_new_grace_versions_and_update_default",
+    author = "rainboyan"
+  )
+  def migration002(implicit db: MongoDatabase): Document = {
+    List(
+      "2022.0.0-RC1",
+      "2022.0.0-RC2",
```

Comment:
Can we exclude all the release candidates and milestones, please?

Reply:
OK